types/react-paginate/index.d.ts

```diff
@@ -129,6 +129,11 @@ export interface ReactPaginateProps {
      */
     hrefBuilder?(pageIndex: number): void;
 
+    /**
+     * Function to set the text on page links. Defaults to `(page) => page`
+     */
+    pageLabelBuilder?: (page: number) => string;
```

Comment:
@reediculous456  What do you think about the errors I have found above? I think they should be fixed.